sound/soc/bcm/allo-katana-codec.c

```diff
@@ -42,8 +42,10 @@
 #define KATANA_CODEC_DEEMPHASIS		(KATANA_CODEC_VIRT_BASE + 6)
 #define KATANA_CODEC_DOP			(KATANA_CODEC_VIRT_BASE + 7)
 #define KATANA_CODEC_FORMAT			(KATANA_CODEC_VIRT_BASE + 8)
-#define KATANA_CODEC_COMMAND		(KATANA_CODEC_VIRT_BASE + 9)
-#define KATANA_CODEC_MAX_REGISTER	(KATANA_CODEC_VIRT_BASE + 9)
+#define KATANA_CODEC_MUTE_STREAM	(KATANA_CODEC_VIRT_BASE + 9)
+#define KATANA_CODEC_COMMAND		(KATANA_CODEC_VIRT_BASE + 10)
```

Comment:
It's your product, so you can do what you want, but by changing the offset of the COMMAND register aren't you creating a forwards- and backwards-compatibility problem? Why not put MUTE_STREAM at offset 10?